packages/translator/src/Charcoal/Translator/Translator.php

```diff
@@ -242,15 +262,32 @@ public function translateChoice($val, $number, array $parameters = [], $domain =
             $locale = $this->getLocale();
         }
```

Comment:
Take advantage of PHP 7 and 8 features.


```suggestion
        $locale ??= $this->getLocale();
```